google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/slow/ITBackupTest.java

```diff
@@ -347,60 +308,58 @@ public void testBackups() throws InterruptedException, ExecutionException, Timeo
     // Test listing operations.
     // List all backups.
     logger.info("Listing all backups");
-    assertThat(instance.listBackups().iterateAll()).containsAtLeast(backup1, backup2);
+    assertThat(instance.listBackups().iterateAll()).contains(backup);
```

Comment:
nit: replace `assertThat` by `assertEquals`, `assertTrue`, ...

Reply:
Done, here and elsewhere in this file